src/test/groovy/org/osiam/test/integration/LostPasswordIT.groovy

```diff
@@ -69,6 +72,19 @@ class LostPasswordIT extends AbstractIT {
         User user = osiamConnector.getUser(userId, accessToken)
         Extension extension = user.getExtension(urn)
         extension.getField('oneTimePassword', ExtensionFieldType.STRING) != null
+
+        //Waiting at least 5 seconds to ensure that an email was send
+        Thread.sleep(5000)
```

Comment:
I know this is a bad idea to use `Thread.sleep` in a test... should I use [CountDownLatch](https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/CountDownLatch.html) instead? Wait until one email was received by greenmail, right?